src/OrbitGl/CallTreeView.h

```diff
@@ -47,6 +48,12 @@ class CallTreeNode {
                                                     std::string module_path,
                                                     std::string module_build_id);
 
+  [[nodiscard]] CallTreeUnwindErrorNode* GetUnwindErrorOrNull(
+      orbit_client_protos::CallstackInfo::CallstackType type);
+
+  [[nodiscard]] CallTreeUnwindErrorNode* AddAndGetUnwindError(
+      orbit_client_protos::CallstackInfo::CallstackType type);
+
```

Comment:
See other comment: the names are way too similar to `GetUnwindErrorsOrNull` and `AddAndGetUnwindErrors`.